src/components/TokenBottomSheet.tsx

```diff
@@ -101,17 +105,26 @@ export function TokenBalanceItemOption({
 function NoResults({
   testID = 'TokenBottomSheet/NoResult',
   searchTerm,
+  selectedFilters,
 }: {
   testID?: string
   searchTerm: string
+  selectedFilters: FilterChip<TokenBalance>[]
 }) {
+  const activeSearchTerms = [searchTerm, ...selectedFilters.map((filter) => filter.name)]
+    .filter((term) => !!term)
+    .join(', ')
```

Comment:
matter of taste: i personally would prefer filters being listed separately. for example:

> No matching results found for "poof" within applied filters: "My tokens", "Popular".